gateway/cli/Cargo.toml

```diff
@@ -13,7 +13,6 @@ description = "CLI tool to control lightning gateway"
 rustc-args = ["--cfg", "tokio_unstable"]
 
 [features]
-default = ["tor"]
```

Comment:
Same here, I guess? Does anyone need to import `gateway-cli` as a cargo dep? I don't think so, but might be wrong.

Reply:
Fixed